app/lib/frontend/templates/views/pkg/admin_page.mustache

```diff
@@ -46,6 +46,61 @@
     You have no verified publisher. Use the <a href="{{& create_publisher_url}}">create publisher</a> page to create one.
   </p>
   {{/user_has_publisher}}
+  {{#show_uploader_admin}}
+  <h3>Uploaders</h3>
+  <div class="mdc-data-table">
+    <table class="mdc-data-table__table" aria-label="Uploaders of package">
+      <thead>
+        <tr class="mdc-data-table__header-row">
+          <th class="mdc-data-table__header-cell email-header" role="columnheader" scope="col">Email</th>
+          <th class="mdc-data-table__header-cell icons-header" role="columnheader" scope="col"></th>
+        </tr>
+      </thead>
+      <tbody class="mdc-data-table__content">
+        {{#uploader_emails}}
+        <tr class="mdc-data-table__row">
+          <td class="mdc-data-table__cell">{{ email }}</td>
+          <td class="mdc-data-table__cell">
+            <a class="-pub-remove-uploader-button"
+               data-email="{{ email }}"
+               title="Remove uploader">&times;</a>
+          </td>
+        </tr>
+        {{/uploader_emails}}
```

Comment:
I wonder if we could show the current invitations?

Reply:
We could probably extend `Consent` to include it as a queryable key for both this and publisher member invites, but that can come later.

Reply:
hmm, part of this could also come should we build audit logs..

Note. we can't expose invitations sent without considering privacy implication.